container/raw/handler.go

```diff
@@ -200,6 +200,17 @@ func (self *rawContainerHandler) GetSpec() (info.ContainerSpec, error) {
 		if utils.FileExists(cpuRoot) {
 			spec.HasCpu = true
 			spec.Cpu.Limit = readInt64(cpuRoot, "cpu.shares")
+			spec.Cpu.Period = readInt64(cpuRoot, "cpu.cfs_period_us")
+			quota := readString(cpuRoot, "cpu.cfs_quota_us")
```

Comment:
because if the quota is notr set, -1 is returned and internally readInt64 uses an uint and this breaks with -1